apps/web/components/booking/pages/BookingPage.tsx

```diff
@@ -264,15 +264,31 @@ const BookingPage = ({
         );
       }
 
+      const query = {
+        isSuccessBookingPage: true,
+        email: bookingForm.getValues("responses.email"),
+        eventTypeSlug: eventType.slug,
+        seatReferenceUid: "seatReferenceUid" in responseData ? responseData.seatReferenceUid : null,
+        ...(rescheduleUid && booking?.startTime && { formerTime: booking.startTime.toString() }),
+      };
+
+      if (eventType.successRedirectUrl) {
+        const url = new URL(eventType.successRedirectUrl);
+        Object.entries(query).forEach(([key, value]) => {
```

Comment:
Add search params related to booking to existing redirect URL that might have it's own query params